app/server/utils/metrics/filters.py

```diff
@@ -147,27 +147,9 @@ def _apply_ca_filters(query, filters, user_join_condition):
     CreditTransfer.transfer_type == TransferTypeEnum.WITHDRAWAL
 ]
 
-
-transaction_volume_filters = [
-    CreditTransfer.transfer_status == TransferStatusEnum.COMPLETE,
-]
-
 standard_payment_filters = [
     CreditTransfer.transfer_status == TransferStatusEnum.COMPLETE,
     CreditTransfer.transfer_type == TransferTypeEnum.PAYMENT,
     CreditTransfer.transfer_subtype == TransferSubTypeEnum.STANDARD
 ]
 
-exchanged_filters = [
-    CreditTransfer.transfer_status == TransferStatusEnum.COMPLETE,
-    CreditTransfer.transfer_type == TransferTypeEnum.EXCHANGE,
-]
-
-beneficiary_filters = [User.has_beneficiary_role == True]
-vendor_filters = [User.has_vendor_role == True]
-
-exhaused_balance_filters = [
-    CreditTransfer.transfer_type == TransferTypeEnum.PAYMENT,
-    TransferAccount._balance_wei == 0
-]
```

Comment:
would these be used in future?

Reply:
I don't imagine they will be.They're pretty self explanatory to rewrite if we do though IMO

Reply:
ok!